REST/PowerShell/Variables/GetVariableInProject.ps1

```diff
@@ -0,0 +1,106 @@
+Function Get-OctopusProject
+{
+    # Define parameters
+    param(
+        $OctopusServerUrl,
+        $ApiKey,
+        $ProjectName
+    )
+
+    # Call API to get all projects, then filter on name
+    $octopusProject = Invoke-RestMethod -Method "get" -Uri "$OctopusServerUrl/api/projects/all" -Headers @{"X-Octopus-ApiKey"="$ApiKey"}
+
+    # return the specific project
+    return ($octopusProject | Where-Object {$_.Name -eq $ProjectName})
+}
+
+Function Get-OctopusProjectVariables
+{
+    # Define parameters
+    param(
+        $OctopusDeployProject,
+        $OctopusServerUrl,
+        $ApiKey
+    )
+
+    # Get reference to the variable list
+    return (Invoke-RestMethod -Method "get" -Uri "$OctopusServerUrl/api/variables/$($OctopusDeployProject.VariableSetId)" -Headers @{"X-Octopus-ApiKey"="$ApiKey"})
+}
+
+    
+
+Function Get-Variable{
+
+    # Define parameters
+    param(
+        $VariableSet,
+        $VariableName
+    )
+    $OutputVariable = "Variable not found based on name and current env"
```

Comment:
Can we default to $null if we don't find a variable with that name and environment combo?